tcp-native/src/main/java/org/ballerinalang/stdlib/tcp/WriteCallbackService.java

```diff
@@ -0,0 +1,64 @@
+package org.ballerinalang.stdlib.tcp;
+
+import io.ballerina.runtime.api.Future;
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelFuture;
+import io.netty.channel.ChannelFutureListener;
+
+/**
+ * WriteCallbackService used to write data via channelPipeline.
+ */
+public class WriteCallbackService {
```

Comment:
I think the better naming would be `WriteFlowControl`

Reply:
+1